README.Rmd

```diff
@@ -8,18 +8,120 @@ output: github_document
 knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>",
-  fig.path = "README/README-"
+  fig.path = "README-"
 )
 ```
 
 # googledrive
 
-[![Build Status](https://travis-ci.org/tidyverse/googledrive.svg?branch=master)](https://travis-ci.org/tidyverse/googledrive)[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/LucyMcGowan/googledrive?branch=master&svg=true)](https://ci.appveyor.com/project/LucyMcGowan/googledrive)[![Coverage Status](https://img.shields.io/codecov/c/github/tidyverse/googledrive/master.svg)](https://codecov.io/github/tidyverse/googledrive?branch=master)
+[![Build Status](https://travis-ci.org/tidyverse/googledrive.svg?branch=master)](https://travis-ci.org/tidyverse/googledrive)[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/tidyverse/googledrive?branch=master&svg=true)](https://ci.appveyor.com/project/tidyverse/googledrive)[![Coverage Status](https://img.shields.io/codecov/c/github/tidyverse/googledrive/master.svg)](https://codecov.io/github/tidyverse/googledrive?branch=master)
 
 `r emo::ji("construction")` WARNING: this is very much under construction `r emo::ji("construction")`
 
-## installation
+## Overview
+
+`googledrive` interfaces with Google Drive from R, allowing users to seamlessly manage files on Google Drive from the comfort of their console `r emo::ji("house")`. 
+
+## Installation
+
 ```{r, eval = FALSE}
+# Obtain the the development version from GitHub:
+# install.packages("devtools")
 devtools::install_github("tidyverse/googledrive")
 ```
 
+## Usage
+
+### Load `googledrive`
+
+```{r}
+library("googledrive")
+```
+
+### Package idiosyncrasies
+
+* All functions begin with the prefix `drive_`
+* Functions and parameters attempt to mimic local file navigating conventions in `R`, such as `list.files`.
+
+### Quick demo
+
+Here's how to list the most recently modified 100 files on your drive. This will kick off your authentication, so you will be sent to your browser to authorize your Google Drive access. The functions here are designed to be pipeable, using `%>%`, however they can also be implemented without.
+
+```{r, eval = FALSE}
+drive_list()
+```
```

Comment:
Do you think it'd be strange to see a bunch of my files? I suppose it doesn't matter